test/jdk/java/awt/event/MouseEvent/ClickDuringKeypress/ClickDuringKeypress.java

```diff
@@ -22,115 +22,100 @@
  */
 
 /*
-  @test 1.2 98/08/05
   @key headful
   @bug 4515763
   @summary Tests that clicking mouse and pressing keys generates correct amount of click-counts
-  @author andrei.dmitriev: area=awt.mouse
   @run main ClickDuringKeypress
 */
```

Comment:
We usually move this block just before a class declaration, for better readability.

Reply:
ok